command-builder/package.json

```diff
@@ -1,19 +1,21 @@
 {
   "name": "@netlify-builder/deploy",
-  "version": "1.0.3",
-  "description": "",
-  "main": "command/index.js",
+  "version": "1.9.0",
```

Comment:
Keep the version the same as previous one.

Reply:
okay